### ISSUE TITLE
[Quest][Classic] Investigate the Blue Recluse/Investigate The Alchemy Shop does not work.

### ISSUE THREAD

**Issue by Rahuum** [dalaranwow/dalaran-wow#5968]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The quest is bugged. Usually the cantation does not spawn any mobs. Even when they do spawn, they instantly escape to the void, unable to be captured by the coffer. The coffer also does not attempt to capture them if placed beforehand, and doesn't last long enough.

**Expected behaviour**: Tell us what should happen instead
Use the Cantation. Four mobs should spawn. Alternatively, use Arcane Explosion to grab one at a time. There should always be mobs available to be grabbed. Kill the mob. It will become stunned and unattackable. Place the coffer down. An animation will play where the coffer captures the rift spawn. The coffer is now lootable with the quest item.

**Steps to reproduce the problem**:

1. Use the cantation. Kill the mobs. Use the cantation again, or arcane explosion. No mobs.
2. Wait a while, use the cantation. Kill a mob. Place the trap. Nothing lootable.
3. Place a trap, kill another mob. Nothing happens.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

https://classic.wowhead.com/quest=1960/investigate-the-alchemist-shop
Includes every step of the quest process and how it's supposed to work.
https://classic.wowhead.com/quest=1960/investigate-the-alchemist-shop#comments:id=3151132 is a particularly good example.

https://www.youtube.com/watch?v=SG8QpIBthXk shows the quest on live classic wow.

**Include Screenshots from the issue if necessary**
